cmd/main.go

```diff
@@ -106,9 +112,10 @@ func main() {
 		grpc.UnaryInterceptor(utils.LogGRPC),
```

Comment:
Can we add [metrics for grpc](https://github.com/Azure/kubernetes-kms/blob/c3180993ec313a0fe18228af10f384c42f35dcb0/cmd/server/main.go#L90) as well?

Reply:
Added metrics for gRPC too. The gRPC request has a different set of attributes compared to the keyvault metric, so I've kept that as a separate metric. The keyvault metric is what will be useful for users to monitor the plugin is functioning correctly.